app/src/main/res/values/strings.xml

```diff
@@ -38,6 +38,25 @@
     <string name="forgot_pass_fragment_button_reset_password">Reset password</string>
     <string name="forgot_pass_fragment_button_verify">Verify</string>
 
+    <!-- VEHICLE VIEW STRINGS -->
+    <string name="home_fragment_2d_mode">2D</string>
```

Comment:
The names don't completely follow the way we agreed to. They are just missing what they are (button, textview....)

Reply:
done